paddle/fluid/inference/tests/book/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ function(inference_test TARGET_NAME)
   endforeach()
 endfunction(inference_test)
 
+####################
+# Inference tests here depend on fluid/tests/book
+# User need to run tests in fluid/tests/book first to generate saved model.
```

Comment:
line 22 `set_tests_properties(test_inference_${TARGET_NAME}${arg} PROPERTIES DEPENDS test_${TARGET_NAME})` already do this depends. So users need not to manually run tests to generated saved model before.

Reply:
But when I ran "ctest -R test_inference_word2vec", the test failed. After I ran "ctest -R test_word2vec", then ran "ctest -R test_inference_word2vec", it passed.
Does the "DEPENDS" really work?

Reply:
When we use `make test` to run all the unittests, this `DEPENDS` can make sure the order. However, if users want to run `ctest -R test_inference_word2vec` separately, he need to run `ctest -R test_word2vec` manually first.

Reply:
Thanks for explain. Updated the comments.